swarm/storage/localstore.go

```diff
@@ -97,13 +97,20 @@ func NewTestLocalStoreForAddr(params *LocalStoreParams) (*LocalStore, error) {
 // After the LDBStore.Put, it is ensured that the MemStore
 // contains the chunk with the same data, but nil ReqC channel.
 func (ls *LocalStore) Put(chunk *Chunk) {
+	if l := len(chunk.SData); l < 9 {
+		log.Debug("incomplete chunk data", "addr", chunk.Addr, "length", l)
+		chunk.SetErrored(ErrChunkInvalid)
+		chunk.markAsStored()
```

Comment:
this method should change name!

Reply:
Yes, that will change in chunk refactor.